evm/src/main/java/org/hyperledger/besu/evm/operation/PushOperation.java

```diff
@@ -49,6 +50,13 @@ public Operation.OperationResult executeFixedCostOperation(
 
   public static OperationResult staticOperation(
       final MessageFrame frame, final byte[] code, final int pc, final int pushSize) {
+    // Handle the PUSH0 special case
+    if(pushSize == 0) {
```

Comment:
Push is the most used instruction in the EVM, hands down, hence this is super hot.  Can this be refactored into a separate Push0Operation?